lib/cli/handle-error.js

```diff
@@ -17,24 +17,33 @@ const { storeLocally: storeTelemetryLocally, send: sendTelemetry } = require('..
 const generateTelemetryPayload = require('../utils/telemetry/generatePayload');
 const resolveErrorLocation = require('../utils/telemetry/resolve-error-location');
 const resolveInput = require('./resolve-input');
+const { legacy, log } = require('@serverless/utils/log');
 
-const consoleLog = (message) => process.stdout.write(`${message}\n`);
-
+// TODO: Remove it with v3 and rename `writeMessageModern` to `writeMessage`
 const writeMessage = (title, message) => {
   let line = '';
   while (line.length < 56 - title.length) {
     line = `${line}-`;
   }
 
   process.stdout.write(' \n');
-  consoleLog(chalk.yellow(` ${title} ${line}`));
-  consoleLog(' ');
+  legacy.consoleLog(chalk.yellow(` ${title} ${line}`));
+  legacy.consoleLog(' ');
 
   if (message) {
-    consoleLog(`  ${message.split('\n').join('\n  ')}`);
+    legacy.consoleLog(`  ${message.split('\n').join('\n  ')}`);
   }
 
-  consoleLog(' ');
+  legacy.consoleLog(' ');
+};
+
+const writeMessageModern = (title, message) => {
+  log.notice(chalk.hex('#fd5750')('\nError:'));
```

Comment:
Output of the final exception information imo qualifies as substantial output, and not event log that's subject to level filtering

Reply:
fully agreed, I already changed that in previous iteration :+1: